applications/crossbar/src/modules/cb_webhooks.erl

```diff
@@ -193,6 +237,8 @@ validate_webhooks(Context, ?HTTP_PATCH) ->
 -spec validate(cb_context:context(), path_token()) -> cb_context:context().
 validate(Context, ?PATH_TOKEN_ATTEMPTS) ->
     summary_attempts(Context, 'undefined');
+validate(Context, ?PATH_TOKEN_SAMPLES) ->
+    fetch_webhook_samples(Context, <<"webhooks_all">>);
```

Comment:
shouldn't this return a list of available webhooks instead of `all` webhooks ?

Reply:
GET `/webhooks` is returning all available webhooks. This one is for fetching all samples in request per what aux team (or ui tema? idk) requested

Reply:
it should be the list of samples not all the samples